maintenance/wikia/WikiFactoryVariables/migrateWikiWordmarks.php

```diff
@@ -37,15 +37,15 @@ public function __construct() {
 		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
 		$this->addOption( 'verbose', 'Show extra debugging output', false, false, 'v' );
 		$this->addOption( 'keyName', 'Key in WikiFactory variable which should be migrated to https', true, true, 'k' );
-		$this->addOption( 'reason', 'Reason to provide when setting a variable', false, true );
+		$this->addOption( 'file', 'File of wiki ids', false, true, 'f' );
 	}
 
 	public function execute() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 6, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[project-id]: wikia:app